### ISSUE TITLE
Frame drops (on collisions ?)

### ISSUE THREAD

**Issue by matthias4217** [agateau/pixelwheels#73]
Hello, I've got a quite old smartphone, but similar games run fine so I guess this could be enhanced. : I experience frame drops, and I guess it's when the player or an enemy collides with something. Usually, the game runs on 60FPS, but it sometimes drops, especially when using the "rifle" weapon or when there are several crashes at once.

**Comment by Wilker-a** [comment]
i can tell that frame drops may also happen when the vehicle is at a high speed, which can be frustrating when trying to do time attacks on the maps, as the frame skips also affects the actual movement for some reason...

**Comment by agateau** [comment]
Hi, do you run the game on Android 10? I am currently investigating bug #130, which looks similar to yours and is likely caused by an issue with the way Android 10 plays sounds. If this is the case then disabling sound effects would make the bug go away (not a real solution, of course, but a good way to confirm the cause)

**Comment by matthias4217** [comment]
I was playing on an older version of Android. (8 or below I believe) Lately, on an other phone (LG G5) with Android 8, I didn't notice the same problem. I've tried it tonight on my LG G5, with Android 11 (LineageOS) and Pixel Wheels 0.20.0. Without sound, it works perfectly. With sounds on, there is indeed a lag, although it doesn't display a frame drop in the debug hud.

I've also tested the version here : https://github.com/agateau/pixelwheels/issues/130#issuecomment-922240631
It works perfectly, so I guess this issue can be closed as well !
